src/panels/lovelace/cards/hui-picture-card.ts

```diff
@@ -86,10 +86,12 @@ export class HuiPictureCard extends LitElement implements LovelaceCard {
           hasHold: hasAction(this._config!.hold_action),
           hasDoubleClick: hasAction(this._config!.double_tap_action),
         })}
-        tabindex="0"
+        tabindex=${this._config.tap_action ? "0" : "-1"}
```

Comment:
```suggestion
        . tabIndex=${this._config.tap_action ? "0" : "-1"}
```

Reply:
`tab_action` can be `none` that should not give it a `tabindex`

Reply:
👍 will use hasAction function

Reply:
Or we go with `isDefined` as Paulus used.

Reply:
Yeah, no reason to set the attribute at all it not necessary 👍